### ISSUE TITLE
Implement Navbar and Footer redesigns

### ISSUE THREAD

**Issue by brenoferreira** [creativecommons/cccatalog-frontend#542]
Designs: https://scene.zeplin.io/project/5dc5a56e81baec82fe6f7851

Implement only navbar and footer.

**Comment by sharmamansi** [comment]
Hey @brenoferreira  cant open the above mentioned link can you tell me the reason?

**Comment by brenoferreira** [comment]
The designs are publicly accessible. Maybe Zeplin was having some issues. Can you try again?

**Comment by sharmamansi** [comment]
Can I work on this?

**Comment by brenoferreira** [comment]
We're currently working on finalising the designs, so this is currently on hold

**Comment by sharmamansi** [comment]
Ohk no problem, however i would like to work on these issues. l will wait wait till designs are finalised please let me know once it is done.

**Comment by coderbabu10** [comment]
sir,can i work on this issue?
